dev-libs/botan/botan-2.6.0.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
+HOMEPAGE="https://botan.randombit.net/"
+COPYRIGHT="1999-2018 The Botan Authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/randombit/botan/archive/$portVersion.tar.gz"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="https://github.com/randombit/botan/archive/$portVersion.tar.gz"
+SOURCE_URI="https://botan.randombit.net/releases/Botan-$portVersion.tgz"
+SOURCE_DIR="Botan-$portVersion"
```
Differences:
* The `SOURCE_DIR` is:
  * `botan-2.6.0` for the tarball on GitHub,
  * `Botan-2.6.0` for the tarball [advertised](https://botan.randombit.net/#release-downloads) on the project's site,

* The contents are identical for all the files, except for `src/build-data/version.txt`:
```diff
diff -u botan-2.6.0/src/build-data/version.txt  Botan-2.6.0/src/build-data/version.txt
--- botan-2.6.0/src/build-data/version.txt	2018-04-10 13:17:29.000000000 +0000
+++ Botan-2.6.0/src/build-data/version.txt	2018-04-10 09:17:29.000000000 +0000
@@ -5,6 +5,6 @@
 release_so_abi_rev = 5
 
 # These are set by the distribution script
-release_vc_rev = None
-release_datestamp = 0
-release_type = 'unreleased'
+release_vc_rev = 'git:d7d080992372cf4fbd569cce1d8cd6aa7599fa0d'
+release_datestamp = 20180410
+release_type = 'release'
```
BTW, if we keep the `SOURCE_URI` (i.e. using the tarball from GitHub) then we would have to add a `SOURCE_FILENAME="botan-$portVersion.tar.gz"`, whereas with the tarball from the project's site we don't need to add that variable. (Well, we need to add `SOURCE_DIR`, but imo having a `release_type = 'release'`in `src/build-data/version.txt` is better than having a `release_type = 'unreleased'` :)